src/components/data-pane/index.tsx

```diff
@@ -47,9 +47,10 @@ export class DataPanelBase extends React.PureComponent<DataPanelProps, {}> {
 
 export const DataPane = connect(
   (state: State) => {
+    const presentUndoableState = state.undoable.present;
     return {
-      data: state.present.dataset,
-      config: state.present.config
+      data: presentUndoableState.dataset,
```

Comment:
We should use a selector for dataset, config for consistency and for reduce the amount of updates needed if we reshape the state in the future.